src/main/scala/no/ndla/imageapi/controller/RawController.scala

```diff
@@ -92,7 +92,8 @@ trait RawController {
             .flatMap(storedRatio => imageRepository.getStoredParametersFor(imageUrl, storedRatio.toString))
             match {
             case Some(storedParameters) => Right(url(converterService.asApiUrl(imageUrl),
-              read[Map[String, String]](write(storedParameters.rawImageQueryParameters)),
+              read[Map[String, String]](write(storedParameters.rawImageQueryParameters.copy(
+                width = intOrNoneWithValidation("width"), height = intOrNoneWithValidation("height")))),
```

Comment:
Fordi vi fra frontenden slår opp på `…/<filnavn>?storedRatio=0.81&focalX=50&focalY=50&ratio=0.81&width=260` og da ønsker vi å beholde `width=260` men forkaste alt annet hvis det finnes parametre for `0.81` i databasen for dette bildet.